tests/foreman/ui_airgun/test_host.py

```diff
@@ -510,3 +524,764 @@ def test_positive_export(session):
+
+    :expectedresults: Host's lifecycle environment and content view match
+        the ones specified in hostgroup.
+
+    :CaseLevel: Integration
+
+    :BZ: 1391656
+    """
+    name = gen_string('alpha').lower()
+    host_name = '{0}.{1}'.format(name, module_host_template.domain.name)
+    lce = make_lifecycle_environment({'organization-id': module_org.id})
+    content_view = make_content_view({'organization-id': module_org.id})
+    ContentView.publish({'id': content_view['id']})
+    version_id = ContentView.version_list({'content-view-id': content_view['id']})[0]['id']
+    ContentView.version_promote({
+        'id': version_id,
+        'to-lifecycle-environment-id': lce['id'],
+        'organization-id': module_org.id,
+    })
+    hostgroup = make_hostgroup({
+        'content-view-id': content_view['id'],
+        'lifecycle-environment-id': lce['id'],
+        'organization-ids': module_org.id,
+    })
+    puppet_proxy = Proxy.list({'search': 'name = {0}'.format(settings.server.hostname)})[0]
+    make_host({
+        'architecture-id': module_host_template.architecture.id,
+        'domain-id': module_host_template.domain.id,
+        'environment-id': module_host_template.environment.id,
+        'hostgroup-id': hostgroup['id'],
+        'location-id': module_loc.id,
+        'medium-id': module_host_template.medium.id,
+        'name': name,
+        'operatingsystem-id': module_host_template.operatingsystem.id,
+        'organization-id': module_org.id,
+        'partition-table-id': module_host_template.ptable.id,
+        'puppet-proxy-id': puppet_proxy['id'],
+    })
+    with session:
+        values = session.host.read(host_name, ['host.lce', 'host.content_view'])
+        assert values['host']['lce'] == lce['name']
+        assert values['host']['content_view'] == content_view['name']
+
+
+@tier2
+def test_positive_inherit_puppet_env_from_host_group_when_create(session, module_org, module_loc):
+    """Host group puppet environment is inherited to host in create
+    procedure
+
+    :id: 05831ecc-3132-4eb7-ad90-155470f331b6
+
+    :customerscenario: true
+
+    :expectedresults: Expected puppet environment is inherited to the form
+
+    :BZ: 1414914
+
+    :CaseLevel: Integration
+    """
+    hg_name = gen_string('alpha')
+    env_name = gen_string('alpha')
+    entities.Environment(name=env_name, organization=[module_org], location=[module_loc]).create()
+    with session:
+        session.hostgroup.create({
+            'host_group.name': hg_name,
+            'host_group.puppet_environment': env_name
+        })
+        assert session.hostgroup.search(hg_name)[0]['Name'] == hg_name
+        values = session.host.helper.read_create_view(
+            {'host.hostgroup': hg_name}, ['host.puppet_environment'])
+        assert values['host']['puppet_environment'] == env_name
+        values = session.host.helper.read_create_view(
+            {'host.inherit_puppet_environment': False},
```

Comment:
what is default value? I am just thinking that maybe it was good idea to use False first and check that `env=''` and then change to True and check that `env=hostgroup.env`. Or behavior is different?

Reply:
The default value is True

Reply:
I feel that test case is build in a wrong way. I explained you and idea and you just resolved comment without any thoughts.

Reply:
was thinking that my response was enough for consideration

Reply:
code added with checking